samples/interop-customvision-textmsg-uwpapp/Documentation/Configuring the IoT Edge Device.MD

```diff
@@ -0,0 +1,108 @@
+# Text Messaging & Custom Vision Interop Samples
+## Progress
+
+- [x] [Introduction](../README.md)  
+- [x] [Step 1 - Setup Development Environment](./Setup%20DevVM.MD)   
+- [x] [Step 2 - Setup Azure Resources](./Setup%20Azure%20Resources.MD)  
+- [x] [Step 3 - Setup Azure IoT Edge for Linux on Windows](./Setup%20Azure%20IoT%20Edge%20for%20Linux%20on%20Windows.MD)
+- [x] [Step 4 - Develop and publish the IoT Edge Linux module](./Develop%20and%20publish%20the%20IoT%20edge%20Linux%20module.MD)  
+- [x] [Step 5 - Create Certificates for Authentication](./Create%20Certificates%20for%20Authentication.MD)  
+- [x] [Step 6 - Develop the Windows C# UWP Application](./Develop%20the%20Windows%20C%23%20UWP%20Application.MD)
+- [ ] **Step 7 - Configuring the IoT Edge Device**  
+- [ ] [Step 8 - Run samples](./Run%20samples.MD) 
+- [ ] [Troubleshooting](./Troubleshooting.MD) 
+---
+
+# Step 7: Configuring the IoT Edge Device.
+In this part we configure the Azure IoT Edge for Linux instance on the IoT edge device.
+
+
+1. **Create Certificates folder inside the EFLOW VM**
+    * Use SSH to log in to the EFLOW VM environment.  
+
+        ```powershell
+        Ssh-EflowVm
+        ``` 
+
+    * Inside the EFLOW VM environment, use mkdir to create the Certificates folder  
+
+        ```powershell
+        mkdir ~/certs/
+        ``` 
+
+2. **Copy Certificates to EFLOW VM environment**
+
+   * Use PowerShell to get the EFLOW VM IP address.  
+
+       ```powershell
+       Get-EflowVmAddr
+       ``` 
+
+    * Use SCP to copy the certificates created in [Step 5](./Create%20Certificates%20for%20Authentication.MD) to the `~/certs` folder of your EFLOW VM environment.  
+
+        ```powershell
+        scp -i 'C:\Program Files\Azure IoT Edge\id_rsa'  .\certs\* iotedge-user@<eflowvm-ip>:~/certs/​
+        ```
+    
+3. **Log in to the Linux environment**
+
+    ```powershell
+    Ssh-EflowVm
+    ```
+
+1. **Set up the Linux firewall**   
+As the Azure Device Client Namespace is used, we use the underlying AMQP protocol for the exchange of messages. We need to specifically allow incoming packets on the ports required by the AMQP protocol. We thus adapt the firewall rules as follows and persist them across reboots:
+    ```bash
+    sudo iptables -A INPUT -p tcp --dport 443 -j ACCEPT 
+    sudo iptables -A INPUT -p tcp --dport 5671 -j ACCEPT 
+    sudo iptables-save | sudo tee /etc/systemd/scripts/ip4save > /dev/null
+    ```
+1. **Read the certificates**  
+    Run the following command to allow Azure IoT Edge to read the certificates.
+    ```bash
+    sudo chown -R iotedge: ~/certs
+    ```
+1. **Provision the Azure IoT Edge for Linux configuration**  
+    To edit config.yaml run the following command:
+    ```bash
+    sudo nano /etc/iotedge/config.yaml
+    ```    
+    * Replace `"<IoT Edge Device connection string>"` in the 'Manual provisioning configuration with the connection string that you obtained in Step 2-6.
```

Comment:
we should mention provisioning here instead of asking the user to manually enter connection string,

Reply:
Added note: Note Skip the following step if Azure IoT Edge was deployed using EFLOW installation. Only replace the connection if it was manually provisioned.